crates/merkle-inclusion/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "bellpepper-merkle-inclusion"
+version = "0.1.0"
+edition = "2021"
+homepage.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+ff = { workspace = true }
+bellpepper-core = { workspace = true }
+bellpepper = { workspace = true }
+bellpepper-keccak = { path="../keccak", version = "0.1.0" }
```

Comment:
should this be a dev-dependency?

Reply:
So far I have implemented thje trait `GadgetDigest` for both `Keccak` and `Sha3` in this crate directly. Which mean I depend on the `bellpepper-keccak` crate to access the hashing algorithm entry point. 

This could be changed if I was to directly implement the trait in `bellpepper-keccak` crate, but I understood that this is not what we wanted.